config/presets/identity/email-password-interaction-policy.json

```diff
@@ -20,18 +20,18 @@
     {
       "@id": "urn:solid-server:email-password-interaction:LoginRenderHandler",
       "@type": "RenderEjsHandler",
-      "RenderEjsHandler:_rootFilePath": {
-        "@id": "urn:solid-server:default:variable:rootFilePath"
+      "RenderEjsHandler:_viewsFolder": {
+        "@id": "urn:solid-server:default:variable:viewsFolder"
       },
-      "RenderEjsHandler:_ejsTemplatePath": "./templates/views/login.ejs"
+      "RenderEjsHandler:_ejsTemplatePath": "./email-password-interaction/login.ejs"
```

Comment:
These templates cannot be used for anything other than the email password interaction because that's the form elements they have available. Not namespacing it would imply that it could be used for some global sense.